packages/test/test-end-to-end-tests/src/test/gc/gcVersionUpgrade.spec.ts

```diff
@@ -226,24 +237,25 @@ describeFullCompat("GC version upgrade", (getTestObjectProvider) => {
     });
 
     afterEach(() => {
+        provider.reset();
```

Comment:
nit: I believe descriptFullCompat already automatically reset on every test?

Reply:
Yes, you are right. Removed it,